MoEngagePluginBase/MoEngagePluginBridge.swift

```diff
@@ -23,7 +23,6 @@ import MoEngageInApps
            let messageHandler = MoEngagePluginMessageDelegate.fetchMessageQueueHandler(identifier: identifier) {
```

Comment:
From where are we constructing the new MoEngageInitConfigCache?.

Reply:
Before the method was called in `MoECapacitorCorePlugin` to create initConfigCache, now it has been moved to Plugin Bridge as discussed. Addressed.